docs/minikube.md

```diff
@@ -0,0 +1,106 @@
+## Minikube
+
+Draft requires a wildcard domain and an ingress controller. Here how to set everything
+up when using [Minikube][Minikube]
+
+### Setup the certificate
```

Comment:
I took the documentation from the installation documentation which was modified by commit https://github.com/Azure/draft/commit/9ff3a31e11a20d28a33ba8d8c8c9f18d26ee6a4f to include the wildcard domain information.